ios/sdk/sdk.xcodeproj/project.pbxproj

```diff
@@ -518,7 +518,7 @@
 				ENABLE_BITCODE = YES;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```

Comment:
Can this line be removed entirely?

Reply:
It can, but running `pod install` puts it back again ☹️ - It was cocoapods that updated these lines rather than any direct action I did. FWIW, the same changes are made to `app.xcodeproj` on `pod install` too, but it didn't seem sensible for me to commit those without any context on the app itself.

Reply:
Hum, then something ese might be necessary, or ese chances are we'll commit the changes again and break the build.

I believe an extra script can be added in the Podfile, React Native has one targeted at supporting M1 builds IIRC.

Reply:
Just to confirm, `pod install` made the change to from `"arm64"` -> `""` which seems like the right thing to do to my mind. If I remove the line (by clearing the value in the Xcode project), `pod install` puts it back to `""`, so I'd say this is all good.

Reply:
Ah that looks good indeed.